src/components/filters.jsx

```diff
@@ -4,10 +4,10 @@ import { useFilters } from '../context/filterContext';
 const Filters = () => {
     const { state, filterDispatch } = useFilters();
   return (
-    <aside className="filters">
+    <form className="filters">
```

Comment:
any specific reason to use form instead of aside?

Reply:
was trying the reset button type for "clear" in filters...switched back to <aside> in latest patch